WatchParty/client/src/components/MoviePage/MoviePage.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Button } from 'react-bootstrap'
+
+import theMovieDb from 'themoviedb-javascript-library'
+
+import './MoviePage.css'
+
+theMovieDb.common.api_key = 'bd5800f4f98c685b042cd33a1a790365'
+theMovieDb.common.base_uri = 'https://api.themoviedb.org/3/'
+
+class MoviePage extends React.Component{
+
+    constructor(props){
+        super(props)
+        
+        this.state={
+            movieItem: [],
+            trailerKey: ""
+        }
+
+        this.fetchMovieData = this.fetchMovieData.bind(this);
+        this.defaultPoster = this.defaultPoster.bind(this);
+    }
+
+    defaultPoster(event){
+        event.target.src = "/default.png"
+    }
+
+    fetchMovieData(){
+        theMovieDb.movies.getById(
+            {"id": this.props.match.params.movieId }, 
+            data => {
+                var response = JSON.parse(data)
+                this.setState({movieItem: response,})
+            }, 
+            err => console.log(err)
+        );
+
+        theMovieDb.movies.getVideos(
```

Comment:
Should we write a mini library just like theMovieDb to interface with our backend API? Might make the code a bit cleaner (since we just basically move the handlers somewhere else)